cmd/helm3.go

```diff
@@ -17,7 +17,8 @@ var (
 	helmVersionRE  = regexp.MustCompile(`Version:\s*"([^"]+)"`)
 	minHelmVersion = semver.MustParse("v3.1.0-rc.1")
 	// See https://github.com/helm/helm/pull/9426
-	minHelmVersionWithDryRunLookupSupport = semver.MustParse("v3.13.0")
+	minHelmVersionWithDryRunLookupSupport  = semver.MustParse("v3.13.0")
+	minHelmVersionWithResetThenReuseValues = semver.MustParse("v3.14.0")
```

Comment:
add some comments, such as the issue link or PR link.

Reply:
Added a ref to the PR where the new flag was introduced :heavy_check_mark: